mesonbuild/interpreter.py

```diff
@@ -2065,6 +2091,121 @@ def get_cross_property_method(self, args, kwargs):
                     }
 
 
+class DependencyFallbacksHolder(InterpreterObject):
+    def __init__(self, interp):
+        InterpreterObject.__init__(self)
+        self.methods.update({'has_function': self.has_function_method,
+                             'find_library': self.find_library_method,
+                             'dependency': self.dependency_method,
+                             'subproject': self.subproject_method,
+                            })
```

Comment:
__[Flake8]__ closing bracket does not match visual indentation ([see on Sider](https://sider.review/gh/repos/19784232/pulls/4595#issue-4884433))

<details>

| Rule |
| - |
| E124 |

</details>
<sub>posted by Sider</sub>